code/modules/projectiles/guns/projectile/sniper.dm

```diff
@@ -182,13 +182,12 @@
 
 /obj/item/projectile/bullet/sniper/accelerator/Range()
 	..()
-	damage += 5
-	if(damage > 40)
-		icon_state = "gaussstrong"
+	damage += 7
```

Comment:
http://www.ss13.eu/tgdb/tg/latest_stats.html#uplinkitems

People only buy the penetrator rounds anyway